### ISSUE TITLE
`setBrowserView` doesn't work as expected

### ISSUE THREAD

**Issue by aadcg** [atlas-engineer/cl-electron#12]
Following a [simple example from the Electron docs](https://www.electronjs.org/docs/latest/api/browser-view):

```lisp
CL-USER> (asdf:load-system :cl-electron)
T
CL-USER> (setf electron::*interface*
               (make-instance 'electron:interface
                              :electron-socket-path (uiop:xdg-runtime-dir "electron.socket")
                              :lisp-socket-path (uiop:xdg-runtime-dir "lisp.socket")))
CL-USER> (electron:launch)
CL-USER> (defvar view (make-instance 'electron:browser-view))
CL-USER> (defvar win (make-instance 'electron:browser-window))
CL-USER> (electron::add-browser-view win view)
"ReferenceError: ID663 is not defined"
NIL
```

Using either `setBrowserView` or `addBrowserView` results in the same error.

**Comment by jmercouris** [comment]
Let me take a look and see what is happening.

**Comment by jmercouris** [comment]
So what that error means is that *somehow* your object is not defined, perhaps it is getting garbage collected or something. The function produces logical code `"ID344.addBrowserView(ID343)"`

**Comment by jmercouris** [comment]
So if we look here, we shall see:

```
ELECTRON> (remote-symbol view)
"ID343"
ELECTRON> (remote-symbol win)
"ID344"
ELECTRON> (format nil "~a.addBrowserView(~a)"
           (remote-symbol win)
           (remote-symbol view))
"ID344.addBrowserView(ID343)"
ELECTRON> (add-browser-view win view)
"ReferenceError: ID343 is not defined"
NIL
ELECTRON> (get-bounds win)
"[object Object]"
NIL
ELECTRON> (focus win)
"undefined"
NIL
```

clearly our window is functional, but our view is not!

**Comment by jmercouris** [comment]
Will investigate view creation.

**Comment by jmercouris** [comment]
Aha!

```
(send-message win "new BrowserView()")
"ReferenceError: BrowserView is not defined"
NIL
```

**Comment by jmercouris** [comment]
Fixed with: e8ba297

please let me know if you have any more problems!

**Comment by jmercouris** [comment]
Nice!!!! Looks so cool!
